modules/nf-core/vizgenpostprocessing/preparesegmentation/main.nf

```diff
@@ -0,0 +1,60 @@
+process VIZGENPOSTPROCESSING_PREPARESEGMENTATION {
+    tag "$meta.id"
+    label 'process_single'
+
+    container 'ghcr.io/wehi-soda-hub/vizgen-postprocessing_container:v0.1.1'
+
+    input:
+    val(meta)
+    path(algorithm_json)
+    path(input_images)
+    val(images_regex)
+    path(um_to_mosaic_file)
+    val(tile_size)
+    val(tile_overlap)
+
+    output:
+    tuple val(meta), path("*.json"), path(input_images), path(algorithm_json), emit: segmentation_files
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        error "VPT is unavailable via Conda. Please use Docker / Singularity / Apptainer / Podman instead."
+    }
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    vpt --verbose \\
+        prepare-segmentation \\
+        $args \\
+        --segmentation-algorithm $algorithm_json \\
+        --input-images "${input_images}/${images_regex}" \\
+        --input-micron-to-mosaic $um_to_mosaic_file \\
+        --output-path . \\
+        --tile-size $tile_size \\
```

Comment:
Are the title size and overlap mandatory ?
If not, they should be passed down through the args statement.